.circleci/config.yml

```diff
@@ -16,6 +16,25 @@ workflows:
                 - master
     jobs:
       - test-behat
+commands:
+  run_test:
+    steps:
+      - run:
+          name: "Run Tests"
+          command: |
+            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 latest
+            composer phpunit
+            WP_MULTISITE=1 composer phpunit
+            rm -rf $WP_TESTS_DIR $WP_CORE_DIR
+            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 nightly true
+            composer phpunit
+            bash bin/full-teardown-wp-tests.sh wordpress_test root '' 127.0.0.1 latest            
+            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 latest
+            vendor/bin/phpunit --group=cli_command
+            WP_MULTISITE=1 vendor/bin/phpunit --group=cli_command
+            rm -rf $WP_TESTS_DIR $WP_CORE_DIR
+            bash bin/install-wp-tests.sh wordpress_test root '' 127.0.0.1 nightly true
+            vendor/bin/phpunit --group=cli_command
```

Comment:
Can we not run these in the initial install of the tests? e.g. rather than running the test install 4 times, we run them each twice and run the `phpunit --group=cli_command` immediately after the default test run?

Reply:
Put simply, _you would really think that would work_, but it does not. Absolutely tried it that way... and every other way Phil and I could come up with, with no luck.